pkg/gadget-context/gadget-context.go

```diff
@@ -371,6 +392,19 @@ func (c *GadgetContext) LoadGadgetInfo(info *api.GadgetInfo, paramValues api.Par
 		}()
 	}
 
+	if c.ExtraInfo() && extraInfo != nil {
+		for k, v := range extraInfo.Data {
+			prefix := strings.Split(k, ".")[0]
+			ei, ok := c.GetVar("extraInfo." + prefix)
```

Comment:
Why do we only take the prefix? Could you add a comment saying what is the syntax of `k`?